components/Collections/examples/CollectionsEditingExample.m

```diff
@@ -100,18 +103,36 @@ - (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView
   // TODO: (shepj) Pending fix for
   // https://github.com/material-components/material-components-ios/issues/1208
   // we must call super which registers classes for supplemental header and footer in editing mode
-  UICollectionReusableView *supplementaryView = [super collectionView:collectionView
-                                    viewForSupplementaryElementOfKind:kind
-                                                          atIndexPath:indexPath];
-  if (supplementaryView) {
-    return supplementaryView;
+  //  UICollectionReusableView *supplementaryView = [super collectionView:collectionView
+  //                                    viewForSupplementaryElementOfKind:kind
+  //                                                          atIndexPath:indexPath];
+  //  UICollectionReusableView *supplementaryView =
+  //  [collectionView dequeueReusableSupplementaryViewOfKind:kind
+  //  withReuseIdentifier:HEADER_REUSE_IDENTIFIER forIndexPath:indexPath]; if (supplementaryView) {
+  //    return supplementaryView;
+  //  }
+
+  NSString *reuseIdentifier;
```

Comment:
I am not sure of the best way to handle this.
An example should not have to deal with custom MDCCollection views.

Should we add a way to query your super class for a reuseIdentifier based on kind?
- (NSString *)reuseIdentifierForKind:(NSString *)kind

We cannot just invoke a super implementation as this method must return a nonnull value.